public/static/js/tour-controller.js

```diff
@@ -64,6 +64,9 @@ dlangTourApp.controller('DlangTourAppCtrl',
 			},
 			'Ctrl-R': function(cm) {
 				$scope.$apply('reset()');
+			},
+			'Ctrl-F': function(cm) {
```

Comment:
`Ctrl-F` is not a good choice, because on most GUI application on Windows and Linux it maps to "Find in page"